### DISCUSSION TITLE
How to backup and restore projects

### DISCUSSION THREAD

**Discussion by Interceptor2025** [community/community#102067] [Projects and Issues]
I noticed there is a "export view data" but opening the file and excel and copying and pasting it back into a new project is slow. Is there a faster way for backing up and restoring projects?

**Comment by chavachai** [comment]
One of the methods involving either forking (independent experimentation), cloning (offline editing), branching (managing versions), or cloud storage (backup) works better. Even better choice is the use of templates (saves time) to make your work easier.

**Reply by Interceptor2025** [reply]
Are you talking about? https://github.com/chavachai?tab=repositories

I meant https://github.com/chavachai?tab=projects

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Interceptor2025 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by Interceptor2025** [comment]
Bump
